src/utils/IdealGraphVisualizer/ServerCompiler/src/com/sun/hotspot/igv/servercompiler/ServerCompilerScheduler.java

```diff
@@ -620,5 +631,65 @@ public void buildUpGraph() {
                 toNode.preds.add(fromNode);
             }
         }
+
```

Comment:
Maybe you could extract the following code into 2 separate methods.

Reply:
Thanks, done!